app/views/attachments/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for(@attachment, :url => document_attachments_path(params[:document_type], params[:document_content_id])) do |f| %>
```

Comment:
Styling on this page doesn't seem right - we are missing the surrounding page elements. In the original app the form was in _form.html.erb and there is extra stuff in the new/edit views.